python/lsst/ts/m2gui/controltab/tab_actuator_control.py

```diff
@@ -33,7 +57,516 @@ class TabActuatorControl(TabDefault):
+
+        Parameters
+        ----------
+        command : enum `CommandActuator`
+            Actuator command.
+        """
+        run_command(self.model.command_actuator, command)
+
+    def _set_widget_and_layout(self):
+        """Set the widget and layout."""
+
+        widget = self.widget()
+        widget.setLayout(self._create_layout())
+
+        # Resize the dock to have a similar size of layout
+        self.resize(widget.sizeHint())
+
+        self.setWidget(self.set_widget_scrollable(widget, is_resizable=True))
+
+    def _create_layout(self):
+        """Create the layout.
+
+        Returns
+        -------
+        layout : `PySide2.QtWidgets.QHBoxLayout`
+            Layout.
+        """
+
+        layout = QHBoxLayout()
+
+        # First column
+        layout_script_control_force_summary = QVBoxLayout()
+        layout_script_control_force_summary.addWidget(
+            self._create_group_script_control()
+        )
+        layout_script_control_force_summary.addWidget(
+            self._create_group_force_summary()
+        )
+
+        layout.addLayout(layout_script_control_force_summary)
+
+        # Second column
+        layout_actuator_control = QVBoxLayout()
+        layout_actuator_control.addWidget(self._create_group_actuator_selector())
+        layout_actuator_control.addWidget(self._create_group_actuator_control())
+
+        layout.addLayout(layout_actuator_control)
+
+        return layout
+
+    def _create_group_script_control(self):
+        """Create the group of script control. The cell control system reads
+        the binary script to perform a seris of actuator motions.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout = QVBoxLayout()
+
+        layout_file = QFormLayout()
+        layout_file.addRow("Script File:", self._info_script["file"])
+        layout_file.addRow("Progress:", self._info_script["progress"])
+
+        layout.addLayout(layout_file)
+
+        layout_control = QGridLayout()
+        layout_control.addWidget(self._buttons_script["load_script"], 0, 0)
+        layout_control.addWidget(self._buttons_script["stop"], 0, 1)
+        layout_control.addWidget(self._buttons_script["resume"], 0, 2)
+
+        layout_control.addWidget(self._buttons_script["run"], 1, 0)
+        layout_control.addWidget(self._buttons_script["pause"], 1, 1)
+        layout_control.addWidget(self._buttons_script["clear"], 1, 2)
+
+        layout.addLayout(layout_control)
+
+        return create_group_box("Script Control", layout)
+
+    def _create_group_force_summary(self):
+        """Create the group of force summary. This is just for the high-level
+        overview of force status.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout = QFormLayout()
+
+        layout.addRow("Minimum Axial (N):", self._labels_force["axial_min"])
```

Comment:
If you use UnitLabel from ts_cRIOpy..CustomLabes, you will get N display after value (as it should), which IMHO looks much better and is easier to read.

Reply:
It is a very good suggestion and I could adapt it in the future's improvement. Right now, I just want to have an available GUI to use the M2 cell as soon as possible. Thanks!